tests/functional/telemetry_data_empty_result/expected-stdout

```diff
@@ -0,0 +1,4 @@
+Connecting to DSN mysql:host=mysql;port=3306;dbname=test;charset=utf8
+Run query "SELECT * FROM `test`.`project_id` WHERE `PROJECT_ID` = '2'"
+Run query "SELECT * FROM `test`.`project_id_stack` WHERE `PROJECT_STACK` = 'connection.north-europe.azure.keboola.com' AND `PROJECT_ID` = '2'"
+Run query "SELECT * FROM `test`.`project_stack` WHERE `PROJECT_STACK` = 'connection.north-europe.azure.keboola.com'"
```

Comment:
Nie je project ID vzdy vyzadovane? Zaujima ma to ohladom bezpecnosti.

Lebo ak sa nejak stane, ze tabulka bude mat preklep v nazve stlpca, tak stiahnes vsetky data - aj ku inym projektom, co by bol pruser .... samozrejme preklep sa moze stat aj v nazve stlpca "stack", ale tam by si stiahol iba to iste ID projektu z viacerych stackov, ... co by bol radovo mensi unik dat.

Inak co sa stane, ak tabulka nema ani jeden zo "specialnych" stlpcov? Nemala by sa vyhodit chyba / alebo ignorovat taku tabulku?

Reply:
tabulka nemusí právě obsahovat `PROJECT_ID` např při seznamu komponent viz:
https://keboola.atlassian.net/browse/COM-452?focusedCommentId=22279

Reply:
překlepy se nesmí stát - do budoucna (v druhé verzi) by se měli stahovat data v určitých projektech pro celou organizaci (ale zatím jen takhle pro projekty ze kterých je extractor spuštěný)

Reply:
ok, ten komentar som si nevsimol, ... ked ste sa tak dohodli tak ok.